### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_ScreenReader: There is no announcement of the output cleared.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9368]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#NVDA;#JAWS;#MAS4.1.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS4.1.3
Screen reader: JAWS, NVDA2019.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Click File then New Notebook.
3. Launch screen reader as NVDA
4. Use tab and navigate to clear results button.

**Actual**
When the code has been executed hit enter on clear results button the data is cleared but no announcement of the same.
Similar fo run cell button

Similar issue with jaws

**Expected:**
There should be an announcement stating the data has been cleared.

**User Impact:**
Screen reader users will not know if the data has been cleared or the cell is running.

**Attachment link for Reference** 
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EY_zpX-t-pFFmKxkvDMxZrYBU4VrkiTwXbP0unhJfSnq0A?e=wrCOvz

**Comment by alanrenmsft** [comment]
dupe of https://github.com/microsoft/azuredatastudio/issues/9404

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by v-jagansai** [comment]
Issue was still repro using NVDA the clear results button was not working.

Attachment added for reference
![Screen Recording (1) (1)](https://user-images.githubusercontent.com/109065315/178463962-8836c6bf-867a-4a94-860a-7a435a95957e.gif)

**Comment by fsteffi** [comment]
Discussed this in office hours with v-tayazd - for actions which execute immediately such as these we shouldn't be announcing that an action is occurring as that tends to be too verbose.

#closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
